core/src/main/java/io/squashql/query/dto/BucketColumnSetDto.java

```diff
@@ -2,29 +2,29 @@
 
 import io.squashql.query.ColumnSet;
 import io.squashql.query.ColumnSetKey;
-import io.squashql.type.TableTypedField;
-import lombok.EqualsAndHashCode;
-import lombok.NoArgsConstructor;
-import lombok.ToString;
-
+import io.squashql.query.Field;
+import io.squashql.query.TableField;
 import java.util.ArrayList;
 import java.util.LinkedHashMap;
 import java.util.List;
 import java.util.Map;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
 
 @ToString
 @EqualsAndHashCode
 @NoArgsConstructor // For Jackson
 public class BucketColumnSetDto implements ColumnSet {
 
-  public String name;
+  public Field name;
 
-  public String field;
+  public Field field;
 
   public Map<String, List<String>> values = new LinkedHashMap<>();
 
-  public BucketColumnSetDto(String name, String field) {
-    this.name = name;
+  public BucketColumnSetDto(String name, Field field) {
+    this.name = new TableField(name); // todo-mde should it really be a table field ? maybe bucketfield ?
```

Comment:
It could be cleaner by I think it is ok. No need to introduce a new type for now.